pulsemeeter/socket/client.py

```diff
@@ -266,17 +266,15 @@ def verify_device(self, device_type, device_id, dev):
         return True
 
     def connect(self, input_type, input_id, output_type, output_id, state=None, latency=None):
-        
+
         if (not self.verify_device(input_type, input_id, 'input') 
                 or not self.verify_device(output_type, output_id, 'output')):
-            return
+                return
 
         command = f'connect {input_type} {input_id} {output_type} {output_id}'
         if state != None: command += f' {state}'
         if latency != None: command += f' {latency}'
 
-        if self.config[input_type][input_id][f'{output_type}{output_id}'] == state:
```

Comment:
why remove this? this is what stops the ui from setting something twice when you run a command outside of it